# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/2d/DrawTargetRecording.cpp
# Commit: a79d9a152a6d
# Full Hash: a79d9a152a6d52a71210d4675d3c86b94615cd1b
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2021-08-04 21:45:54
# Regressor Bug: 1722300
# File Overlap Count: 1
# Description:
#   Bug 1722300 - patch 4 - Implement internal destinations when generating PDF output through cairo. r=mattwoodrow
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D120960
# ==============================================================================

diff -r b11f7f6bff04 -r a79d9a152a6d gfx/2d/DrawTargetRecording.cpp
--- a/gfx/2d/DrawTargetRecording.cpp	Wed Aug 04 12:52:37 2021 +0000
+++ b/gfx/2d/DrawTargetRecording.cpp	Wed Aug 04 12:52:38 2021 +0000
@@ -202,6 +202,11 @@
   mRecorder->RecordEvent(RecordedLink(this, aDestination, aRect));
 }
 
+void DrawTargetRecording::Destination(const char* aDestination,
+                                      const Point& aPoint) {
+  mRecorder->RecordEvent(RecordedDestination(this, aDestination, aPoint));
+}
+
 void DrawTargetRecording::FillRect(const Rect& aRect, const Pattern& aPattern,
                                    const DrawOptions& aOptions) {
   EnsurePatternDependenciesStored(aPattern);